docker/Dockerfile

```
@@ -0,0 +1,5 @@
+# Import Node base image (change version as needed)
+FROM node:12-slim
+
+#Install the node SDK
+RUN npm install ibm-watson
```

Still need a newline at end of file

```suggestion
RUN npm install ibm-watson

```